lib/tools/settings-client-commands.js

```diff
@@ -64,7 +64,7 @@ commands.requireRunningSettingsApp = async function requireRunningSettingsApp (o
 
   log.debug('Starting Appium Settings app');
   const {
-    timeout = 5000,
+    timeout = this.adbExecTimeout,
```

Comment:
I don't think it is a good idea to mix adb exec timeout this this one. Consider customizing it on a higher level (e.g. appium-android-driver)

Reply:
You mean creating another capability, and using that here?
As I understand it needs changes both here and appium-android-driver.
I can do it, but as it is two different pull requests, I need a capability name and an okay from core maintainers.